identity.go

```diff
@@ -50,7 +50,9 @@ func newIdentity(cfg config, userIDString string) (identity, error) {
 		return identity{}, err
 	}
 
-	payload := append(epubSignKey, userID...)
+	payload := make([]byte, len(epubSignKey) + len(userID))
+	copy(payload, epubSignKey)
+	copy(payload[len(epubSignKey):], userID)
```

Comment:
Whats wrong with `append` ?

Reply:
If, for whatever reason, `epubSignKey` happens to have enough capacity to hold `userID`, then not only will you get `payload == epubSignKey` but you will also consequently be mutating `epubSignKey`